srcpkgs/zeal/template

```diff
@@ -1,20 +1,43 @@
 # Template file for 'zeal'
 pkgname=zeal
 reverts="20141123_1 20141123_2"
-version=0.6.1
+version=0.7.2
 revision=1
 build_style=cmake
-configure_args="CONFIG+=no_libappindicator"
+build_helper=$(vopt_if qt5 qmake qmake6)
+configure_args="-DZEAL_RELEASE_BUILD=ON -DCMAKE_DISABLE_FIND_PACKAGE_X11=$(vopt_if x11 no yes)
+		$(vopt_bool qt5 ZEAL_USE_QT5)"
 hostmakedepends="pkg-config extra-cmake-modules"
-makedepends="qt5-webkit-devel xcb-util-keysyms-devel libarchive-devel qt5-x11extras-devel sqlite-devel"
-depends="bsdtar"
+makedepends="$(vopt_if x11 xcb-util-keysyms-devel) libarchive-devel sqlite-devel"
+depends="desktop-file-utils hicolor-icon-theme"
 short_desc="Simple offline API documentation browser"
 maintainer="Farhad Shahbazi <grauwolf@geekosphere.org>"
 license="GPL-3.0-or-later"
 homepage="https://zealdocs.org/"
 distfiles="https://github.com/zealdocs/zeal/archive/v${version}.tar.gz"
-checksum=095c08f9903071849d5c79878abd48237ce1615f16d324afff1873ab6b5f0026
+checksum=9c4ce9ab5700985f4c562e995301f98767f14cd045f047dee2520ec4ce1855cf
 
-if [ "$CROSS_BUILD" ]; then
-	hostmakedepends+=" qt5-host-tools qt5-qmake"
+build_options="x11 qt5"
+build_options_default="x11"
+
+# qt6-webengine is unavailable on 32-bit systems
```

Comment:
the more correct way to target platforms supported by qt6-webengine is

https://github.com/void-linux/void-packages/blob/0369ba676946964f1353b98225c774f6dd7062ad/srcpkgs/kdepim-addons/template#L25-L27